glide.yaml

```diff
@@ -49,9 +49,11 @@ import:
   subpackages:
   - semver
 - package: github.com/coreos/ignition
-  version: v0.19.0
+  version: 28aea78861d53a9b3a9c2b5cf51305e00f36fa14
```

Comment:
I required https://github.com/coreos/ignition/pull/501, otherwise, we have to copy paste the Append code out of Ignition, because none of the packages had appropriate types. Once there is a release, definitely planning to bump to that.

Reply:
Gotcha. There should be v0.21.0 this week.

Reply:
This definitely won't be ready until after then. I'm kinda nudging it along in spare time. I'll plan to bump to that version.